index.html

```diff
@@ -30,5 +30,82 @@ <h2>I’am a Software Developer</h2>
+        <ul class="tags">
+            <li>css</li>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+        </ul>
+        <button type="button">See Project</button>
+    </div>
+    <div class="cards">
+        <div class="card">
+            <div class="card_container">
+                <div class="card_title">Profesional Art Printing Data</div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="tags">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <button  type="button">See Project</button>
+        </div>
+        <div class="card">
+            <div class="card_container">
+                <div class="card_title">Profesional Art Printing Data</div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="tags">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <button  type="button">See Project</button>
+        </div>
+        <div class="card">
+            <div class="card_container">
+                <div class="card_title">Profesional Art Printing Data</div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="tags">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <button  type="button">See Project</button>
+        </div>
+        <div class="card">
+            <div class="card_container">
+                <div class="card_title">Profesional Art Printing Data</div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="tags">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <button type="button">See Project</button>
+        </div>
+    </div>    
+</div>
+<div class="aboutme">
+    <div class="aboutme_title">About me</div>
+    <p class="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <div class="aboutme_btn_img">
```

Comment:
Kindly. each part of your website should be a SECTION tag, this will give a better structure to your site